src/containers/forms/PlanForm/style.css

```diff
@@ -0,0 +1,11 @@
+.close {
+  color: #6c757d !important;
+  font-size: 30px;
+}
+
```

Comment:
@moshthepitt can we add some margin-bottom to these buttons?

<img width="631" alt="Screen Shot 2019-08-16 at 10 13 48 AM" src="https://user-images.githubusercontent.com/4141527/63185361-add4f280-c00e-11e9-82ad-eb0265e2acb1.png">

Reply:
Yes! Will do

Reply:
![activities](https://user-images.githubusercontent.com/372073/63192855-bbbd5e80-c074-11e9-91c5-638a713f03c9.png)

 done